### DISCUSSION TITLE
[Bug] "Quick fix" => "Fix Using Copilot" errors with  "command 'inlineChat.start' not found"

### DISCUSSION THREAD

**Discussion by MarArMar** [community/community#77756] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

When hovering over a problem, if I left click and Click "Quick fix" I get 2 options : 

- Explain using Copilot => Works
- Fix Using Copilot : opens a modal "command 'inlineChat.start' not found"

GitHub Copilot v1.139.0
GitHub Copilot Chat v0.10.2
GitHub Copilot Labs v0.17.1121

VSCode Version: 1.84.2
Electron: 25.9.2
ElectronBuildId: undefined
Chromium: 114.0.5735.289
Node.js: 18.15.0
V8: 11.4.183.29-electron.0
OS: Windows_NT x64 10.0.19043

**Comment by SonnyTaylor** [comment]
I would suggest trying to reinstall Copilot, if that doesnt work maybe vscode might need an update?

**Reply by MarArMar** [reply, marked as answer]
Thanks for your answer. I see other people having the same problem they use like me a fork of VSCode it seems it is because of that : 

https://github.com/getcursor/cursor/issues/1026
